server/integration_tests/test_data/fulfillment_api_sdg/chart_config.json

```diff
@@ -1109,57 +1109,7 @@
   ],
   "relatedThings": {
     "childPlaces": {},
-    "childTopics": [
-      {
-        "dcid": "dc/topic/sdg_1.1",
-        "name": "End Poverty by 2030",
-        "types": [
-          "Topic"
-        ]
-      },
-      {
-        "dcid": "dc/topic/sdg_1.2",
-        "name": "Reduce Poverty by Half by 2030",
-        "types": [
-          "Topic"
-        ]
-      },
-      {
-        "dcid": "dc/topic/sdg_1.3",
-        "name": "Social Protection Systems",
-        "types": [
-          "Topic"
-        ]
-      },
-      {
-        "dcid": "dc/topic/sdg_1.4",
-        "name": "Equal Rights and Access to Basic Services by 2030",
-        "types": [
-          "Topic"
-        ]
-      },
-      {
-        "dcid": "dc/topic/sdg_1.5",
-        "name": "Reduce Extreme Climate Events And Disasters By 2030",
-        "types": [
-          "Topic"
-        ]
-      },
-      {
-        "dcid": "dc/topic/sdg_1.a",
-        "name": "Resource Mobilization",
-        "types": [
-          "Topic"
-        ]
-      },
-      {
-        "dcid": "dc/topic/sdg_1.b",
-        "name": "Poverty Eradication Policy Frameworks",
-        "types": [
-          "Topic"
-        ]
-      }
```

Comment:
Ohh, this is a problem for SDG topics which aren't in the KG!  So we should not do existence check if `dc=sdg`